app/reducers/optional.js

```diff
@@ -1,7 +0,0 @@
-//this is a simple pass thru that offers an extension point
```

Comment:
this implies there is always a `index.js` and `initialState.js` file in reducers. The `initialState.js` file would by default export an empty POJO object.